src/components/Playbooks/Runs/Submit/PlaybookSelectResource.tsx

```diff
@@ -9,53 +15,80 @@ type PlaybookSelectResourceProps = {
 export default function PlaybookSelectResource({
```

Comment:
I think this should be a generic ResourceSelector dropdown, with the selectors pass in from the playbook spec